### PR TITLE
Chien/refactor

### PR DESCRIPTION
**Pull request by ChienNQuang**
forgive my soul

### CODE REVIEW

**Review by StarryFolf — APPROVED** [review]
god bless your soul

File: src/Api/IApiMarker.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+namespace Api;
+
+public interface IApiMarker
+{
+
```
Comment (Vzart) [inline src/Api/IApiMarker.cs:5]: What is this for?

File: src/Application/Common/Exceptions/ConflictException.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+namespace Application.Common.Exceptions;
+
+public class ConflictException : Exception
+{
+    public ConflictException(string message) : base(message)
+    {
+    }
+}
```
Comment (Vzart) [inline src/Application/Common/Exceptions/ConflictException.cs:8]: what type of exception that we listed as ConfilctException?

File: src/Application/Users/Commands/CreateUser/CreateUserCommandValidator.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        RuleFor(x => x.LastName)
+            .NotEmpty().WithMessage("Last name is required.")
+            .MaximumLength(50).WithMessage("Last name cannot exceed 50 characters.");
+
+        RuleFor(x => x.Position)
+            .NotEmpty().WithMessage("Position cannot be empty.")
+            .MaximumLength(64).WithMessage("Position cannot exceed 64 characters.");
+    }
+
+    private static bool BeNotAdmin(string role)
```
Comment (kaitoz11) [inline src/Application/Users/Commands/CreateUser/CreateUserCommandValidator.cs:40]: Why not `IsNotAdmin`

**Review by kaitoz11 — APPROVED** [review]
Pray for your soul

**Review by vzart — APPROVED** [review]
Your contribution is vital and at utmost importance to the survival of us, I thank thee.

File: src/Api/IApiMarker.cs (hunk 1 above)
Comment (ChienNQuang) [reply src/Api/IApiMarker.cs:5]: for WebApplicationFactory used in test projects to find its assembly

File: src/Application/Common/Exceptions/ConflictException.cs (hunk 2 above)
Comment (ChienNQuang) [reply src/Application/Common/Exceptions/ConflictException.cs:8]: we try to update/add a resource which has a key constraint and that resource violates it

File: src/Application/Users/Commands/CreateUser/CreateUserCommandValidator.cs (hunk 3 above)
Comment (ChienNQuang) [reply src/Application/Users/Commands/CreateUser/CreateUserCommandValidator.cs:40]: Is with Must?

**Pull request merged**